packages/@contentful--app-scripts/lib/upload/create-zip-from-directory.js

```diff
@@ -0,0 +1,61 @@
+const glob = require('glob');
+const chalk = require('chalk');
+const fs = require('fs');
+const JSZip = require('jszip');
+const { showCreationError } = require('../utils');
+
+async function convertPathToFileData(filePath) {
+  return new Promise((res) => {
+    fs.readFile(filePath, 'utf8', (err, data) => {
+      res([filePath, data]);
+    });
+  });
```

Comment:
You can use `fs/promises` instead of wrapping. Although, being a script, also sync API would do